src/projectscene/view/clipsview/clipcontextmenumodel.cpp

```diff
@@ -33,11 +33,12 @@ void ClipContextMenuModel::load()
     updateStretchEnabledState(*enableStretchItem);
 
     auto colorItems = makeClipColourItems();
+    m_colorMenu = makeMenu(muse::TranslatableString("clip", "Clip color"), colorItems);
```

Comment:
we don't store items in the model, we used findItem from the abstract menu model